e2e/api/public/api.js

```diff
@@ -0,0 +1,40 @@
+/* eslint func-names:0 */
```

Comment:
Yes, I hesitated ... At the same time it's a good practice, but it's a pain in tests ...
@djhi @jpetitcolas an opinion ?